src/textual_summary/icon_or_image.jsx

```diff
@@ -1,11 +1,22 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
+import './styles.scss';
 
 /**
  * Render icon or an image with a title
  */
 export default class IconOrImage extends React.Component {
-  renderIcon = () => <i className={this.props.icon} title={this.props.title} />;
+  renderIcon = () => {
+    if (this.props.background && this.props.background !== '') {
```

Comment:
Maybe add `this.props.background.trim() !== ''` to the condition. To avoid strings containing only white spaces.